vpr/src/place/place.cpp

```diff
@@ -269,6 +274,8 @@ static void free_fast_cost_update();
 
 static double comp_bb_cost(e_cost_methods method);
 
+static double comp_layer_bb_cost(e_cost_methods method);
```

Comment:
Comment this.

Reply:
I will comment the functions related to calculating net cost in the other PR. In that PR, I have moved all these functions in another file.